packages/gitbook/src/components/DocumentView/Table/utils.ts

```diff
@@ -12,6 +17,23 @@ export function getRecordValue<T extends number | string | boolean | string[] |
     return record.values[definitionId];
 }
 
+/**
+ * Get the covers for a record card.
+ * Returns both the light and dark covers.
+ * The light cover is a string or a content ref (image or files column type).
+ * The dark cover is a content ref (image column type).
+ */
+export function getRecordCardCovers(record: DocumentTableRecord, view: DocumentTableViewCards) {
```

Comment:
```suggestion
export function getRecordCardCovers(record: DocumentTableRecord, view: DocumentTableViewCards): { light: ContentRefFile | ContentRefURL | null; dark: ContentRefFile | ContentRefURL | null } {
```

Better to be explicit in the typing